src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -218,40 +218,73 @@ export default class TaskRoleContainerList extends React.Component {
-      res.push('This job does not contain SSH info.');
-      res.push(
-        'Please note that if your docker image does not have openssh-server and curl packages, SSH will not be enabled.\n',
-      );
-      res.push(
-        'Solution 1: Use one of the recommended docker images on the submission page.',
-      );
-      res.push(
-        'Solution 2: Use your own image, but enable SSH for it. Please follow the instructions on https://aka.ms/AA5u4sq to do such work.',
-      );
-      this.setState({
-        monacoProps: { value: res.join('\n') },
-        monacoTitle: `SSH to ${id}`,
-      });
+    if (config.launcherType !== 'k8s') {
+      if (!containerSshInfo) {
+        const res = [];
+        res.push('This job does not contain SSH info.');
+        res.push(
+          'Please note that if your docker image does not have openssh-server and curl packages, SSH will not be enabled.\n',
+        );
+        res.push(
+          'Solution 1: Use one of the recommended docker images on the submission page.',
+        );
+        res.push(
+          'Solution 2: Use your own image, but enable SSH for it. Please follow the instructions on https://aka.ms/AA5u4sq to do such work.',
+        );
+        this.setState({
+          monacoProps: { value: res.join('\n') },
+          monacoTitle: `SSH to ${id}`,
+        });
+      } else {
+        const res = [];
+        res.push('# Step 1. Open a Bash shell terminal.');
+        res.push('# Step 2: Download the private key:');
+        res.push(
+          `wget '${sshInfo.keyPair.privateKeyDirectDownloadLink}' -O ${sshInfo.keyPair.privateKeyFileName}`,
+        );
+        res.push('# Step 3: Set correct permission for the key file:');
+        res.push(`chmod 600 ${sshInfo.keyPair.privateKeyFileName}`);
+        res.push('# Step 4: Connect to the container:');
+        res.push(
+          `ssh -i ${sshInfo.keyPair.privateKeyFileName} -p ${containerSshInfo.sshPort} root@${containerSshInfo.sshIp}`,
+        );
+        res.push('');
+        this.setState({
+          monacoProps: {
+            value: res.join('\n'),
+            options: {
+              wordWrap: 'off',
+              readOnly: true,
+            },
+          },
+          monacoTitle: `SSH to ${id}`,
+        });
+      }
     } else {
       const res = [];
-      res.push('# Step 1. Open a Bash shell terminal.');
-      res.push('# Step 2: Download the private key:');
-      res.push(
-        `wget '${sshInfo.keyPair.privateKeyDirectDownloadLink}' -O ${sshInfo.keyPair.privateKeyFileName}`,
-      );
-      res.push('# Step 3: Set correct permission for the key file:');
-      res.push(`chmod 600 ${sshInfo.keyPair.privateKeyFileName}`);
-      res.push('# Step 4: Connect to the container:');
-      res.push(
-        `ssh -i ${sshInfo.keyPair.privateKeyFileName} -p ${containerSshInfo.sshPort} root@${containerSshInfo.sshIp}`,
-      );
-      res.push('');
+      if ('ssh' in containerPorts) {
```

Comment:
pls check whether `userssh` is enabled in ssh plugin, 'ssh' port is always there

Reply:
Done.